script/Connector-Axelar.s.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {ConnectorAxelarRouter} from "src/routers/axelar/Router.sol";
+import {CentrifugeConnector} from "src/Connector.sol";
+import {RestrictedTokenFactory, MemberlistFactory} from "src/token/factory.sol";
+import "forge-std/Script.sol";
+
+// Script to deploy Connectors with an Axelar router.
+contract ConnectorAxelarScript is Script {
+    // address(0)[0:20] + heccak("Centrifuge")[21:32]
+    bytes32 SALT = 0x000000000000000000000000000000000000000075eb27011b69f002dc094d05;
```

Comment:
Can you explain why we have a constant `SALT`? And what why the encoded meaning behind it is significant here?

Reply:
Yeah there's no documentation on this right now: https://github.com/foundry-rs/book/issues/782

The `salt` itself leads to deploying contracts to the same address on different chains.